### DISCUSSION TITLE
Enable selecting text in the theme previewer

### DISCUSSION THREAD

**Discussion by lykahb** [ghostty-org/ghostty#2967] [Feature Requests, Ideas] [answered]
After choosing a theme, a user has to re-type its name in the config. I'd like to select and copy the name instead.

The left side with the list of themes reacts to clicks, so it might be difficult to make text selectable there. Selecting text on the main preview pane should be easier. It has the name of the theme on top. Another, less important upside is that the preview pane would be able to show the colors of the selected text.

**Comment by jcollie** [comment, marked as answer]
As shown in the help, typing `c` will copy the theme name to the clipboard.
![image](https://github.com/user-attachments/assets/d56bd546-9c41-496c-9522-f36a37895102)

**Reply by lykahb** [reply]
Thanks! That wasn't obvious to me. I think the previewer could improve the UX:
1. The previewer could show a few most common actions on the bottom, similar to `htop`. Invoking help and copying a theme name are likely the best choices.
2. The order of actions in the help can be sorted by how common or hard to discover they are. Quitting and navigating the list up/down are rather intuitive. Keybinding to invoke the help window isn't helpful much - user already invoked it.
